support-data-rest/src/main/java/uk/ac/ebi/uniprot/api/literature/output/LiteratureEntryFilter.java

```diff
@@ -0,0 +1,84 @@
+package uk.ac.ebi.uniprot.api.literature.output;
+
+import uk.ac.ebi.uniprot.common.Utils;
+import uk.ac.ebi.uniprot.domain.citation.Journal;
+import uk.ac.ebi.uniprot.domain.literature.LiteratureEntry;
+import uk.ac.ebi.uniprot.domain.literature.builder.LiteratureEntryBuilder;
+import uk.ac.ebi.uniprot.search.field.LiteratureField;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author lgonzales
+ * @since 2019-07-04
+ */
+public class LiteratureEntryFilter {
```

Comment:
Need better approach to unset the fields with null. I am looking into it.

Reply:
ok